app/assets/stylesheets/layout.scss

```diff
@@ -2137,6 +2137,30 @@ table {
     form {
       margin-top: $line-height / 2;
     }
+
+    .responses-count {
+      .far {
+        @extend .fa-minus-square;
```

Comment:
Unfortunately font-awesome doesn't provide mixins nor placeholder selectors for these styles. We could write:

```scss
      .far::before {
        content: fa-content($fa-var-minus-square);
      }
```

But, to be honest, this feels like copying the body of a method instead of calling the method. I'd rather use `@extend`, even if we aren't following the rules :relieved:. @Senen What you decide is fine with me :wink:.